crates/flipperzero/src/alloc.rs

```diff
@@ -0,0 +1,55 @@
+use core::alloc::{GlobalAlloc, Layout};
+use core::ffi::c_void;
+
+use flipperzero_sys::c_string;
+use flipperzero_sys::furi;
+
+extern "C" {
+    #[link_name = "free"]
```

Comment:
Since the Rust function name and the symbol names are identical, using `#[link_name]` isn't strictly necessary.